modules/installation-gcp-service-account-credentials.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-account.adoc
+
+[id="installation-gcp-service-account-credentials_{context}"]
+= Providing the GCP service account credentials
+
+{product-title} requires a Google Cloud Platform (GCP) service account.
+
+.Prerequisites
+
+* You created a service account for your cluster and generated its key.
+
+.Procedure
+
+. Create a new service account in the project that you use to host your
+{product-title} cluster. See
+link:https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating_a_service_account[Creating a service account]
+in the GCP documentation.
+
+. Grant the service account the appropriate permissions. You can either
+grant it the individual permissions that follow or assign it the `Owner` role.
```

Comment:
You can probably remove both instances of _it_ to read more smoothly. 
_You can either grant the individual permissions that follow or assign the `Owner` role.